internal/backend/db/dbgorm/projects_test.go

```diff
@@ -70,7 +70,7 @@ func TestGetProjects(t *testing.T) {
 	assert.ErrorIs(t, err, gorm.ErrRecordNotFound)
 
 	// test getProjectByName after delete
-	_, err = f.gormdb.getProject(f.ctx, p.Name)
+	_, err = f.gormdb.getProjectByName(f.ctx, p.Name)
```

Comment:
getProject knows how to recognize whether it's name or ID. Not anymore?

Reply:
this is getProject, doesn't look like it works with either name or id
plus, id is now uuid and not a string 
```go
func (db *gormdb) getProject(ctx context.Context, projectID sdktypes.UUID) (*scheme.Project, error) {
	return getOne(db.db, ctx, scheme.Project{}, "project_id = ?", projectID)
}
```